cves/CVE-2017-17081.yml

```diff
@@ -64,7 +64,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  
+
+  The library does not properly authetnicate widths and heights of a passed-in
```

Comment:
```suggestion
  The library does not properly authenticate widths and heights of a passed-in
```

Reply:
Agreed with "validate", fixed in upcoming commit.